spec/client_spec.rb

```diff
@@ -308,6 +308,31 @@
     end
   end
 
+  describe "#locked?" do
+
+    context "when lock is available" do
+      after(:each) { lock_manager.unlock(@lock_info) if @lock_info }
+
+      it "returns false" do
+        res = lock_manager.locked?(resource_key)
```

Comment:
I know redlock-rb's test suite is not 100% up to "modern" idiomatic rspec, but I think in this case it could benefit from a little refactor. Today I'd probably write this spec as

```rb
describe '#locked?' do
  subject { lock_manager.locked?(resource_key) }
  
  context 'when lock is available' do
    it { is_expected.to eq false }
  end
  
  # ... 2nd context with before and after
end
```

It's a bit a question of personal taste, but as said, IMO the test gains a bit in terms readability here.